examples/angular/src/app/components/content/content.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container>
+  <div *ngIf="account">
+    <button (click)=signOut()>Log out</button>
+    <button (click)=switchProvider()>Switch Provider</button>
+  </div>
+  <app-form *ngIf="account" [account]="account" (onAddMessage)="onSubmit($event)"></app-form>
+  <app-messages *ngIf="!!account && !!messages.length" [messages]="messages"></app-messages>
+</ng-container>
```

Comment:
I'm a complete novice with Angular but are we able to just do a single `ngIf` on the container for `account`? Then `app-messages` can just check `messages.length`?

Reply:
Sure it can work that way too, I just left the markup this way to keep it somehow separated!